marlowe-dashboard-client/src/Marlowe/Client.purs

```diff
@@ -20,6 +20,13 @@ import Marlowe.Semantics (Input, MarloweData, MarloweParams, Slot, TransactionEr
 import Plutus.Contract.StateMachine (InvalidTransition, SMContractError)
 import Wallet.Types (ContractError)
 
+-- These are the endpoints of the main marlowe (control) contract.
+-- TODO: provide encoding for these and use that with the `invokeEndpoint` call in Capability.Marlowe
+data MarloweAppEndpoint
+  = Create
```

Comment:
I don't see anywhere in this PR that calls `SetLastMarloweAppEndpointCall` with either `Create` or `Redeem`, are those missing?

Reply:
Huh, that certainly sounds like a mistake. Well spotted. I'll have to check properly in the morning...